module-4/backend/week_four.md

```diff
@@ -5,15 +5,26 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's your favorite project management tool?
+  I prefer waffle to pivotal
 2. Why do we create staging environments?
+  To keep unfinished products away from production, so that bugs/errors/and precompilation issues can be solved without effecting the live product. 
 3. What are the characteristics of a good README (in your opinion)?
+  Explaination of hwo to start up the app, examples of code and explainations of structure of the app, the problems it aims to solve, and maybe even some history on the project and the impetus for its creation. 
```

Comment:
yup! everything a new person may need to know to contribute to your code or everything an employer might want to know about the code to learn more about the project